github/prci.py

```diff
@@ -231,6 +299,8 @@ def main():
     parser = create_parser()
     args = parser.parse_args()
 
+    pid_file_handling()
```

Comment:
It handles the pid file creation but it also does the pruning. 

I suggest to rename this function focusing on its core behaviour, e.g: `prune_old_jobs` or `prune_orphan_jobs`.

Reply:
PID Handling is the early beginning of all applications, and later decisions are taken afterward.

Based in working of prci, the  "core behaviour" is to create the pid file, if there isn't  PID file, start the application, if not, clean the environment, update the PID file and then start prci.

Reply:
Function called is named `pid_file_handling`, one can suppose that it just handles the PID file, but it does more than that, so its name must be changed for something clearer.

Reply:
Armando, create a PID is part of the start process, and cleaning the vms and process is the next step, they are connected activities.

Function is very straight forward, and you will take no more than 10 minutes to understand this logic

Reply:
I got it, it's about the semantics, when I read the name of the function it should be clear what it is suppose to do without reading its implementation.